src/carousel/carousel.ts

```diff
@@ -183,6 +195,10 @@ export class NgbCarousel implements AfterContentChecked,
     }
   }
 
+  showNavigation(): boolean { return !this.hideNavigation || (this.slides != null && this.slides.length > 1); }
```

Comment:
@wilhelmhb could you please remove the ` this.slides.length > 1` check (and the corresponding tests(s) based on https://github.com/ng-bootstrap/ng-bootstrap/issues/2274#issuecomment-381173512 ?

Reply:
If I remove the `this.slides.length > 1` check, then the navigation items will be hidden, no matter how many slides you have.
If you have several slides, IMHO, it would never make sense to hide these navigations items. Hence the check.